tools/UnityPaths/UnityPaths.cs

```diff
@@ -18,7 +18,7 @@ public static class Paths
 
         // Adjust the root based on Windows or MacOS. Linux is currently unsupported.
         public static string ImprobableUnityRootPath =>
-            RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? "/Applications" : Path.Combine(@"c:\", "Unity");
+            RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? "/Applications/Unity" : Path.Combine(@"c:\", "Unity");
```

Comment:
That breaks how it's currently working locally. Why was this change made?

Reply:
This will otherwise install all unity version at `/Applications/2018.2.0b10` `/Applications/2018.2.0f2` etc